### PR TITLE
Add OS Activate and OS Verify Operations

### PR DESCRIPTION
**Pull request by asafsonnv**

### CODE REVIEW

File: os/os.go [hunk 1]
```diff
@@ -170,3 +170,53 @@ func (i *InstallOperation) Execute(ctx context.Context, c *internal.Clients) (*o
 	}
 	return installResp, nil
 }
+
+// ActivateOperation represents the parameters of a Activate operation.
```
Comment (greg-dennis) [inline os/os.go:174]: Can you move this above InstallOperation so we keep the operations alphabetized?

**Review by greg-dennis — CHANGES_REQUESTED** [review]
Thanks!

**Comment by greg-dennis** [comment]
Sorry for the delay. I was away on vacation last week. Can you just add a few single unit tests to os_test.go? These are simple, unary operations, so they would look like TestTime: https://github.com/openconfig/gnoigo/blob/main/system/system_test.go#L312

**Comment by asafsonnv** [comment]
Hi sorry I accidentally did force push. can we revert it? or should I create a new MR with all the changes?

**Pull request closed without merging**